easybuild/easyconfigs/n/nodejs/nodejs-16.15.1-GCCcore-11.3.0.eb

```diff
@@ -2,7 +2,7 @@ easyblock = 'ConfigureMake'
 
 name = 'nodejs'
 version = '16.15.1'  # LTS on 2022-06-12
-local_libversion = '93'
+_libversion = '93'
```

Comment:
@lexming also here - why the change?

Reply:
I systematically remove `local` from custom variables in easyconfigs since the change allowing `_variable` as name. It's a better format.